### DISCUSSION TITLE
Allow for pinning scores from ingame webpage

### DISCUSSION THREAD

**Discussion by theredpro-sj** [ppy/osu#30913] [Ideas]
Pretty simple, just allow users to pin scores from directly inside the game client. Maybe a ui could be brought up to pin if you right click a score as that currently does nothing.
<img width="1470" alt="Screenshot 2024-11-29 at 12 17 36 AM" src="https://github.com/user-attachments/assets/41c44b18-cf11-4607-9ffe-ea72d32b7386">

**Comment by peppy** [comment]
@ppy/team-web do we have an endpoint for toggling score pins?

**Reply by nanaya** [reply]
not on the api yet
